package.json

```diff
@@ -18,9 +18,9 @@
   },
   "license": "BSD-2-Clause",
   "dependencies": {
-    "acorn": "^7.1.1",
+    "acorn": "^7.2.0",
     "acorn-jsx": "^5.2.0",
-    "eslint-visitor-keys": "^1.1.0"
+    "eslint-visitor-keys": "github:eslint/eslint-visitor-keys"
```

Comment:
Flagging this just so we don't forget to update it to the new version once we release `eslint-visitor-keys`.